require-lib.php

```diff
@@ -34,3 +28,15 @@ function jetpack_require_lib( $slug ) {
 	}
 	trigger_error( "Cannot find a library with slug $slug.", E_USER_ERROR );
 }
+
+function jetpack_require_lib( $slug ) {
```

Comment:
This needs to be renamed back to `require_lib`, otherwise this function is getting declared twice.